README.md

```diff
@@ -107,6 +108,49 @@ env:
   - DEPLOY_WORKER=true
 ```
 
+## Releasing
+
+1. Checkout a new release branch from `develop`.
+2. Bump the version of each of the subpackages (frontend, backend, worker).
+3. Stage changes for subpackage `package.json` and `package-lock.json`.
+4. Bump the version of the main package.
+   - You will need to use `--force` as the working directory is not clean.
+5. Push both the version commit and tag to GitHub.
+6. Open a new pull request and consolidate changelist from the commits made since the previous release.
+7. Merge pull request into master to trigger Travis build and deploy.
+   - Manually trigger a custom build on Travis if there were changes made to the worker.
+
+**Example:**
+
+```bash
+# Create a new release branch for a new minor version (e.g. v1.6.0 -> v1.7.0)
+$ git checkout develop
+$ git checkout -b release-v1.7.0
+
+# Bump frontend version
+$ cd frontend/
+$ npm version minor
+
+# Bump backend version
+$ cd ../ && cd backend/
+$ npm version minor
+
+# Bump worker version
+$ cd ../ && cd worker/
+$ npm version minor
+
+# Stage changes for subpackage package.json and package-lock.json
+$ cd ../
+$ git add -A
```

Comment:
I'm wary about `git add -A` because it may add more than just the `package.*` files if you have stray files in your local repository.

Reply:
```suggestion
$ git add */package.json */package-lock.json
```
Good point. Would this be better?